assets/views/home/modEdit.ejs

```diff
@@ -47,7 +47,7 @@
         <label class="col-sm-3 control-label">Introduction:</label>
 
         <div class="col-md-8">
-          <textarea class="form-control" ng-model="refUser.intro"></textarea>
+          <textarea class="form-control" ng-model="refUser.intro" ng-maxlength="10000" maxlength="10000"></textarea>
```

Comment:
Could we add a max length counter here as well?

Reply:
Done